osu.Game/Beatmaps/BeatmapInfo.cs

```diff
@@ -51,6 +51,11 @@ public class BeatmapInfo : IEquatable<BeatmapInfo>, IJsonSerializable, IHasPrima
         [NotMapped]
         public BeatmapOnlineInfo OnlineInfo { get; set; }
 
+        /// <summary>
+        /// The length in milliseconds of this beatmap's song.
```

Comment:
this is incorrect

Reply:
i don't understand why this is incorrect?, if it's because of `beatmap's song.` then not sure what's correct: because locally (in song select), the length property is set to a calculation of `lastObject.EndTime - firstObject.StartTime`, and in online, the length property is set to a retrieved length from an api response (which i believe is the length of the song itself), do you have a better xmldoc for this?

Reply:
Correct. The song could be any length, and is not dependent on hitobjects.

How about "the playable length of this beatmap"?